scripts/generate_openapi_client.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+openapi_filename=$(dirname $0)/../openapi/smart-events.yaml
+output_path=$(dirname $0)/../openapi/generated
+package_name="smart-events"
+additional_properties="ngVersion=6.1.7,npmName=${package_name},supportsES6=true,withInterfaces=true,withSeparateModelsAndApi=true,modelPackage=model,apiPackage=api"
+
+npx @openapitools/openapi-generator-cli version-manager set 5.4.0
+echo "Generating SDKs"
+
+# validate the openapi spec for smart-events
+
+echo "Validating OpenAPI ${openapi_filename}"
+npx @openapitools/openapi-generator-cli validate -i "$openapi_filename"
+
+echo "Clearing existing client"
+
+# remove old generated models
+
+rm -Rf $output_path/model $output_path/api
+
+# generate an API client for smart-events
+
+echo "Generating source code based on ${openapi_filename}"
+
+npx @openapitools/openapi-generator-cli generate -g typescript-axios --api-name-suffix "sdk" -i \
```

Comment:
`--api-name-suffix "sdk"` is to have file names `bridges-apisdk.ts` instead of `bridges-apiapi.ts`

Reply:
Uhm do you think we should drop the suffix entirely?
I suggested to remove "API" from tags just to align to other services. But I think we could eventually drop the suffix entirely if it's possible. Wdyt?

Reply:
@kelvah I see your point. In my updates of sandbox openapi annotation this was included. It seems not in Jacopo's version of annotations. So I asked him https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/436/files#r832319124